src/configs/config.example.json

```
@@ -61,6 +61,11 @@
         "allowedGuilds": [],
         "blockedGuilds": [],
         "allowedUsers": [],
+        "areaRestrictions": {
```

I can see why you have it setup like this but I don't think it's very intuitive when compared to the rest of the config. We aren't asking admins to make changes to the keys anywhere else, just the values. I think breaking it up like the role permissions are would make more sense to the average admin that doesn't want to read the codebase to figure out what you're supposed to do. 

```suggestion
        "areaRestrictions": {
           "rule1": {
              "roles": []
              "areas": []
          },
           "rule2": {
              "roles": []
              "areas": []
          }
     }
```

---

I would left current version and make documentation for this feature instead.
I've been trying to find cases in my mind, and setting more than one role to same areas list would make sense only in multi-guild cases.
With unusable keys, configuration would grow way bigger than intended unless we'll find usage for them like.. displaying access ID on WebUI or something to identify future problems. 
I'm open for discussion around that ,)